apps/server/src/shared/domain/entity/boardnode/types/board-do.builder.ts

```diff
@@ -1,3 +1,4 @@
+import { CollaborativeTextEditorElement } from '@shared/domain/domainobject/board/collaborative-text-editor-element.do';
```

Comment:
Combine with `Element` imports below.

Reply:
You shouldn't use `@shared` inside of shared, use a relative link here.